docs/testing/e2e-testing-standards.md

```diff
@@ -0,0 +1,254 @@
+End-to-end (e2e) tests are costly. Typically, they are expensive in terms of resources, take much effort to get going and are very sensitive to changes of any sort in the flow.Even small breaks anywhere along the line mean e2e tests cannot continue--they become brittle and nearly unmanageable over time.
```

Comment:
if they are so bad, why do we then use them?

Reply:
Better to write what benefits you can get from the UI e2e tests and the purpose of it

Reply:
Sorry for that. Modified now